tests/interfaces/test_batch_tensorflow.py

```diff
@@ -36,7 +36,8 @@ def test_jacobian_options(self, mocker, tol):
 
         dev = qml.device("default.qubit", wires=1)
 
-        with tf.GradientTape() as t:
+        with tf.GradientTape(watch_accessed_variables=False, persistent=True) as t:

```

Comment:
@maliasadi I might suggest not using `persistent=True` in the interface tests, just to ensure that PennyLane still supports the (default) non-persistent mode. From my experience in the past, it is very easy for changes to the internal PL logic to break non-persistent mode.

Reply:
Thanks @josh146! I addressed this concern. But, not totally removed the use cases of `persistent=True`. This is due to this fact that using `experimental_use_pFor=False` plays a crucial rule to get the performance and removing `persistent=True` yields not using the former. Now, we have test cases both with and without these options. Let me know if you think differently to remove all the remaining cases.